templates/index

```diff
@@ -233,6 +233,7 @@ cisco_wlc_ssh_show_cdp_neighbors_detail.template, .*, cisco_wlc_ssh, sh[[ow]] c[
 cisco_wlc_ssh_show_ap_config_general.template, .*, cisco_wlc_ssh, sh[[ow]] ap con[[fig]] ge[[neral]]
 cisco_wlc_ssh_show_ap_summary.template, .*, cisco_wlc_ssh, sh[[ow]] ap sum[[mary]]
 cisco_wlc_ssh_show_sysinfo.template, .*, cisco_wlc_ssh, sh[[ow]] sysi[[nfo]]
+cisco_wlc_ssh_show_inventory.template, .*, cisco_wlc_ssh, sh[[ow]] inve[[ntory]]
```

Comment:
Do you mind moving the name template up one line to adhere to our index file standards. After that, I can get this merged in.